dashboard/templates/base.html

```diff
@@ -42,6 +42,17 @@
         {% else %}
         "logout": "",
         {% endif %}
+        {% if current_user_courses_info%}
+            "user_courses_info":'{{current_user_courses_info|safe}}',
```

Comment:
after the user login to the course this variable will be populated like below `"user_courses_info":'[{"course_id":"556114","course_name":"LATIN 842 009 SU 2062"},{"course_id":"717406","course_name":"DENT 877 005 SU 2067"}]'`